docs/core/tutorials/netcore-hosting.md

```diff
@@ -0,0 +1,312 @@
+
+This article gives an overview of the steps necessary to start the .NET Core runtime from native code, create an initial AppDomain, and execute managed code in it.
+
+## Prerequisites
+
+Because hosts are, themselves, native applications, this tutorial will cover constructing a C++ application to host .NET Core. You will need a C++ development environment (such as that provided by [Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48146)).
+
+You will also want a simple .NET Core application to test the host with, so you should install the [.NET Core SDK and CLI](https://www.microsoft.com/net/core) and build a small .NET Core test app (such as a 'Hello World' app).
+
+This tutorial and its [associated sample](https://github.com/dotnet/docs/tree/master/samples/core/hosting) build a Windows host, but please see the notes at the end of this article about hosting on Unix.
+
+## Creating the host
+
+A sample host demonstrating the steps outlined in this article is available in our [.NET Core samples](https://github.com/dotnet/docs/tree/master/samples/core/hosting). Comments in the sample's host.cpp file clearly associate the numbered steps from this tutorial with where they are performed in the sample.
+
+Please bear in mind that the sample host is meant to be used for learning purposes so it is light on error checking and bears a design that emphasizes readability over efficiency. More real-world host samples are available in the [dotnet/coreclr](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts) repository. The [CoreRun host](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts/corerun), in particular, is a good general-purpose host to study after reading through the simpler sample.
+
+### A note about mscoree.h
+The primary .NET Core hosting interface (`ICLRRuntimeHost2`) is defined in [MSCOREE.IDL](https://github.com/dotnet/coreclr/blob/master/src/inc/MSCOREE.IDL). A header version of this file (mscoree.h - which your host will need to reference) is produced via MIDL when the [.NET Core runtime](https://github.com/dotnet/coreclr/) is built. If you do not want to build the .NET Core runtime, mscoree.h is also available as a [pre-built header](https://github.com/dotnet/coreclr/tree/master/src/pal/prebuilt/inc) in the dotnet/coreclr repository. 
+
+### Step 1 - Identify the managed entry point
+After referencing necessary headers ([mscoree.h](https://github.com/dotnet/coreclr/tree/master/src/pal/prebuilt/inc/mscoree.h) and stdio.h, for example), one of the first things a .NET Core host must do is locate the managed entry point it will be using.
```

Comment:
To make sure I understand, you're saying I should leave out the parenthetical bit?